test/images/volume/nfs/run_nfs.sh

```diff
@@ -22,21 +22,22 @@ function start()
     while getopts "G:" opt; do
         case ${opt} in
             G) gid=${OPTARG};;
+            *):;;
```

Comment:
[SC2220](https://github.com/koalaman/shellcheck/wiki/SC2220) - _"Invalid flags are not handled. Add a *) case."_

This just adds a default 'true' case that does nothing, alternative could be to just disable the check.